scripts/zkai_attention_kv_fuller_crossing_grid_gate.py

```
@@ -0,0 +1,617 @@
+        key: value
+        for key, value in result.items()
+        if key not in {"mutation_results", "mutations_checked", "mutations_rejected"}
+    }
+    validate_core(core)
+    expected = build_base_result()
+    if core != expected:
+        raise FullerCrossingGridGateError("checked result drift")
+    if mutation_results is None and mutations_checked is None and mutations_rejected is None:
+        return
+    if [item.get("name") for item in mutation_results] != list(EXPECTED_MUTATION_NAMES):
+        raise FullerCrossingGridGateError("mutation name drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)